src/bun.js/webcore/streams.zig

```diff
@@ -525,9 +525,19 @@ pub const StreamResult = union(Tag) {
     into_array: IntoArray,
     into_array_and_done: IntoArray,
     pending: *Pending,
-    err: Syscall.Error,
+
+    err: union(Err) {
```

Comment:
maybe do

```
Error: Syscall.Error
JSValue: JSC.JSValue
```

That JSvalue version also needs to be kept alive somehow.

Reply:
Fixed on https://github.com/oven-sh/bun/pull/4127/commits/4d07c988db7ed2a783567aa8fb07dec1bfd4dda3